importer/SeFornminSv.py

```diff
@@ -48,10 +48,18 @@ def set_descriptions(self):
 
     def set_raa(self):
         """
-        All items have a 'raa-nr'.
+        All items have a 'raa-nr' and an 'id'.
+        The latter is used to link to the
```

Comment:
It would be nice to have a blank line between the first line of the documentation and the rest. This is also in accordance with PEP257